### ISSUE TITLE
Uniquely save decrypted .vm2 cards

### ISSUE THREAD

**Issue by retrosapien** [bucanero/apollo-ps3#46]
I was backing up some ps2 .vme cards to .vm2, and learning how to add old saves to the ps3.
I came across this small inconvenience.

When decrypting  ps2 classic saves, they are saved with a default name. 
I was was having to constantly rename them between user accounts, to stop them from being over written. 

Could they instead be saved with the name or id of the use? Or in folders named after the user or id?
Or even if they were saved in the home folder, with the originals?

for example;
/dev_hdd0/savedata/vmc/USER0.VM2
/dev_hdd0/savedata/vmc/000000xx/APOLLO0.VM2
/dev_hdd0/home/000000xx/ps2emu2_savedata/PS2U10000/APOLLO0.VM2

If it helps, although I sure you already know this, the user name can be found found here;
/home/000000xx/localusername 
its just a txt file.
If the owners.xml file helps with this, i am sorry. I have not figured out how or what that does yet.

An option to save the decrypted cards renamed to .ps2 could help many people as well.
Although it is not a priority as the dev of mymc-dual has already made it possible to open .vm2 cards without the need to rename to .ps2 before. 
=)

**Comment by bucanero** [comment]
yes, the filename is currently hard-coded, but I guess it would make more sense to just use `original_file_name` + `.VM2`
it was hard-coded because it was a quick feature to add, and I assumed that most users would just focus on PS3 saves.

probably along with that change I should change the hard-coded filenames from the import/encrypt option too

**Comment by retrosapien** [comment]
I think saving to a subfolder named after the user number would be the easiest and most effective approach. 

tl;dr
Original file name won't work either when dealing with multiple users. The ps2classics vme name is hardcoded too I think. So you would still have the same filename collision if you don't rename it before changing users. 

And I wouldn't bother changing the the import/encryption either. For the same reason. 
Unless ps2classics has a custom named vme option that I've missed?

For ps3 saves, the xmb is ok for most saves. Apollo is more helpful for locked and protected save, right? Or saves from other consoles? 
Although I do like having one app to rule them all. 
=)

**Comment by bucanero** [comment]
Now Apollo v2.0.0 can open and manage .VME files directly, import/export single saves, decrypt, encrypt back, and many other things, so probably this is no longer needed.

Feel free to check the latest version and comment back if there's still an issue.
